data.sql

```diff
@@ -1,8 +1,15 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES
-(1, 'Agumon', '2020-02-03', 0, TRUE, 10.23),
-(2, 'Gabumon', '2018-11-15', 2, TRUE, 8),
-(3, 'Pikachu', '2021-01-07', 1, FALSE, 15.04),
-(4, 'Devimon', '2017-05-12', 5, TRUE, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+('Agumon', 'February 3, 2020', 0, TRUE, 10.23),
+('Gabumon', 'November 15, 2018', 2, TRUE, 8),
+('Pikachu', 'January 7, 2021', 1, FALSE, 15.04),
+('Devimon', 'May 12, 2017', 5, TRUE, 11),
+('Charmander', 'February 8, 2020', 0, TRUE, 11),
+('Plantmon', 'November 15, 2021', 2, TRUE, 5.7),
+('Squirtle', 'April 2, 1993', 3, TRUE, 12.13),
+('Angemon', 'June 12, 2005', 1, FALSE, 45),
+('Boarmon', 'June 7, 2005', 7, TRUE, 20.4),
+('Blossom', 'October 13, 1998', 3, TRUE, 17),
+('Ditto', 'May 14, 2022', 4, TRUE, 22);
```

Comment:
Some animals have a weight as negative as mentioned in the project requirements. Consider updating that part.
![image](https://user-images.githubusercontent.com/33268927/183878706-9aad8fb2-968c-4641-bae5-7610776caa95.png)

Reply:
Hi @muneebulrehman,

Thank you for your suggestion. I have made the changes. https://github.com/NayanKaran/sql-demonstration/pull/2/commits/e9b4584c1e441125fdb0ea0c5ff169b5368fed52